### PR TITLE
app-backup/rear: bump to v2.4

### PR DESCRIPTION
**Pull request by teissler**
Closes: https://bugs.gentoo.org/664004

Signed-off-by: Timo Eissler <morlix@morlix.de>

### CODE REVIEW

**Comment by mgorny** [comment]
I'm afraid I can't test it right now. Could you ping me when the download location works again?

```
>>> Downloading 'https://download.sourceforge.net/project/rear/rear/2.4/rear-2.4.tar.gz'
--2018-08-25 08:06:42--  https://download.sourceforge.net/project/rear/rear/2.4/rear-2.4.tar.gz
Resolving download.sourceforge.net (download.sourceforge.net)... 52.85.131.174, 52.85.131.81, 52.85.131.254, ...
Connecting to download.sourceforge.net (download.sourceforge.net)|52.85.131.174|:443... connected.
GnuTLS: A TLS fatal alert has been received.
GnuTLS: received alert [40]: Handshake failed
Unable to establish SSL connection.
!!! Couldn't download 'rear-2.4.tar.gz'. Aborting.
```

**Comment by teissler** [comment]
I changed the SRC_URI to github instead of SF, please restart your tests.

File: app-backup/rear/rear-2.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Relax-and-Recover is a setup-and-forget bare metal disaster recovery solution"
+HOMEPAGE="http://relax-and-recover.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```
Comment (mgorny) [inline app-backup/rear/rear-2.4.ebuild]: Are you sure about that? I've just looked at [random file](https://github.com/rear/rear/blob/master/usr/sbin/rear) and it says GPL-3+.

File: app-backup/rear/rear-2.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	net-fs/nfs-utils
+	sys-apps/gawk
+	sys-apps/iproute2
+	sys-apps/lsb-release
+	sys-apps/util-linux
+	sys-apps/sed
+	sys-block/parted
+	sys-boot/syslinux
+	virtual/cdrtools
+	udev? ( virtual/udev )
```
Comment (mgorny) [inline app-backup/rear/rear-2.4.ebuild]: Please sort conditions lexically too.

File: app-backup/rear/rear-2.4.ebuild (hunk 1 above)
Comment (teissler) [reply app-backup/rear/rear-2.4.ebuild]: I changed the License.

File: app-backup/rear/rear-2.4.ebuild (hunk 2 above)
Comment (teissler) [reply app-backup/rear/rear-2.4.ebuild]: Done

File: app-backup/rear/rear-2.4.ebuild (hunk 1 above)
Comment (mgorny) [reply app-backup/rear/rear-2.4.ebuild]: Why `GPL-3` and not `GPL-3+`?

**Comment by mgorny** [comment]
Also please squash your commits.

File: app-backup/rear/rear-2.4.ebuild (hunk 1 above)
Comment (teissler) [reply app-backup/rear/rear-2.4.ebuild]: My mistake sorry, i fixed it.

**Comment by teissler** [comment]
Squashed

File: app-backup/rear/rear-2.4.ebuild (hunk 2 above)
Comment (mgorny) [reply app-backup/rear/rear-2.4.ebuild]: I'm sorry for not being clear here. First all unconditionals, sorted, then conditions, sorted ;-).

Comment (teissler) [reply app-backup/rear/rear-2.4.ebuild]: Done

**Pull request closed without merging**